src/ServiceControl.Config/Commands/InstallerCompatibility.cs

```diff
@@ -0,0 +1,35 @@
+﻿namespace ServiceControl.Config.Commands
+{
+    using System;
+    using System.Threading.Tasks;
+    using Framework;
+
+    static class InstallerVersionCompatibilityDialog
+    {
+        public static async Task<bool> ShowValidation(Version instanceVersion, Version installerVersion, IServiceControlWindowManager windowManager)
+        {
+            var instanceIsNewer = instanceVersion > installerVersion;
+            var installerOfDifferentMajor = instanceVersion.Major != installerVersion.Major;
+
+            if (instanceIsNewer)
+            {
+                await windowManager.ShowMessage(
```

Comment:
Following @DavidBoike comment, we could use `hideCancel:true` to show only the `OK` button here.

Reply:
Added `hideCancel: true`